controllers/admin/AdminAjaxPsAccountsController.php

```diff
@@ -184,4 +186,25 @@ public function ajaxProcessGetOrRefreshAccessToken()
             SentryService::captureAndRethrow($e);
         }
     }
+
+    /**
+     * @return void
+     *
+     * @throws Exception
+     */
+    public function ajaxProcessGetContextQuery()
+    {
+        try {
+            /** @var QueryBus $queryBus */
+            $queryBus = $this->module->getService(QueryBus::class);
+
+            header('Content-Type: text/json');
+
+            $this->ajaxRender(
+                (string) json_encode($queryBus->handle(new GetContextQuery()))
+            );
+        } catch (Exception $e) {
+            SentryService::captureAndRethrow($e);
```

Comment:
Is Sentry very useful ?
We catch the exception, so we returns nothing has response ?

Reply:
I think we should return a good json reponse in case of rethrow (with error code 500 and an error property)

Reply:
I agree